src/database.ts

```diff
@@ -1833,14 +1838,20 @@ class Database extends common.GrpcServiceObject {
   restore(backupPath: string): Promise<RestoreDatabaseResponse>;
   restore(
     backupPath: string,
-    options?: CallOptions
```

Comment:
Oh interesting. I've gone back and forth on this being very indecisive. I think what you're suggesting may be a bit better than my previous attempts to not make this a breaking change. Would you be able to give us feedback on https://github.com/googleapis/nodejs-spanner/pull/1282#discussion_r540401803? I think this can work.